docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,353 @@
+Hover over the icon to see more details in a tooltip.
+
+// What you might do with this info
+This gives you an overview of how much time is spent (and how that time is spent) loading resources.
+This high-level information may not help you diagnose a problem on its own, but it could act as a
+signal to look at more granular information in the <<synthetics-analyze-one-step-network>> section.
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-timing.png[Network timing visualization for one step in a browser monitor in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-one-step-metrics]]
+=== Metrics
+
+// What info it includes
+The Metrics visualization gives you insight into the performance of the web page visited in
+the step and what a user would experience when going through the current step.
+
+* *First contentful paint (FCP)* focuses on the initial rendering and measures the time from
+  when the page starts loading to when any part of the page's content is displayed on the screen.
+* *Largest contentful paint (LCP)* measures loading performance. To provide a good user experience,
+  LCP should occur within 2.5 seconds of when the page first starts loading.
+* *Cumulative layout shift (CLS)* measures visual stability. To provide a good user experience,
+  pages should maintain a CLS of less than 0.1.
+* *`DOMContentLoaded` event (DCL)* is triggered when the browser completes parsing the document.
+  Helpful when there are multiple listeners, or logic is executed:
+  `domContentLoadedEventEnd - domContentLoadedEventStart`.
+* *Transfer size* represents the size of the fetched resource. The size includes the response header
+  fields plus the response payload body.
+
+[NOTE]
+====
+Largest contentful paint and Cumulative layout shift are part of Google's
+https://web.dev/vitals/[Core Web Vitals], an initiative that introduces a set of metrics
+that help categorize good and bad sites by quantifying the real-world user experience.
+====
+
+Next to each metric, there's an icon that indicates whether the value is
+higher (image:images/icons/sortUp.svg[Value is higher icon]),
+lower (image:images/icons/sortDown.svg[Value is lower icon]),
+or the same (image:images/icons/minus.svg[Value is the same])
+compared to all runs over the last 24 hours.
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-metrics.png[Metrics visualization for one step in a browser monitor in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-one-step-object]]
+=== Object weight and count
+
+++++
+<div style="color:#F04E98">
+++++
+
+// What info it includes
+The *Object weight* and *Object count* visualization shows...
+
+// What you might do with this info
+Use this information to...
+
+++++
+</div>
+++++
+
+// Screenshot of the viz
+[role="screenshot"]
+image::images/synthetics-analyze-one-step-object.png[Object visualization for one step in a browser monitor in the Synthetics UI]
```

Comment:
@paulb-elastic can you help me understand the purpose of the "Object weight" and "Object count" visualizations for individual steps? 

![image](https://user-images.githubusercontent.com/10479155/223186373-05b72c87-2d2e-4003-be0d-5fd64aa8ee8f.png)

Reply:
@colleenmcginnis the _weight_ is the amount (in bytes/KB etc.), whereas the _count_ is the number/quantity of them.

This helps give a different analysis, e.g. you might have a large number of JS files, each of which will need a separate download, but they may be collectively small.  This could be an opportunity to combine them.

Does that help?  LMK if it needs more detail.